app/views/collections/stock-sales.md

```diff
@@ -9,3 +9,26 @@
 [WGBH Stock Sales](http://wgbhstocksales.org/) is the clip licensing division of the WGBH Media Library and Archives. Stock Sales licenses originally shot WGBH footage and web content to television producers, publishers and professional media organizations. The Stock Sales website features over 10,000 clips and 12 WGBH collections that users can search, browse, and license.
 
 Search and browse thousands of WGBH clips available to license! Click on the thumbnails below to screen examples:
+
+## Examples
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
+
+<div class="document col-md-4 col-sm-6">
+    <a href="http://wgbhstocksales.org/catalog/TODO">
+        <img src="TODO">
+        <div class="info">Title goes here</div>
+    </a>
+</div>
```

Comment:
@afred: sorry that it's not clear from the screenshot: this is content under a tab. (And since there is only one tab, we don't show the tab title element in the UI.)

This is a grid of items that's designed to look identical to the grids of items that can be returned by a search. Rather than creating new machinery for just one page, I decided it was more efficient just to cram the html we wanted, where we wanted it, on this one page.